src/main/java/com/yana/internship/entity/Order.java

```diff
@@ -0,0 +1,83 @@
+package com.yana.internship.entity;
+
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "app_order")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @CreationTimestamp
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "creation_date")
+    private Date creationDate;
+
+    @Temporal(TemporalType.DATE)
+    @Column(name = "check_in_date")
+    private Date checkInDate;
+
+    @Temporal(TemporalType.DATE)
+    @Column(name = "check_out_date")
+    private Date checkOutDate;
+
+    @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
```

Comment:
Should we remove Apartments if the Order will be removed?

Reply:
Changed to CascadeType.Merge